controller.go

```diff
@@ -119,7 +120,15 @@ type driverData struct {
 	capability driverapi.Capability
 }
 
+type ipamData struct {
+	config    ipamapi.Config
+	allocator ipamapi.Allocator
+	// default address spaces are provided by ipam driver at registration time
+	defaultLocalAddressSpace, defaultGlobalAddressSpace string
```

Comment:
minor:
May be we should just have single `defaultAddressSpace` field, and based on driver's scope we can initialize it at time of registration? This is assuming that IPAM driver operates in one mode at a time.